pty/src/unix.rs

```diff
@@ -119,7 +119,14 @@ impl PtyFd {
 
     fn get_size(&self) -> Result<PtySize, Error> {
         let mut size: winsize = unsafe { mem::zeroed() };
-        if unsafe { libc::ioctl(self.0.as_raw_fd(), libc::TIOCGWINSZ, &mut size as *mut _) } != 0 {
+        if unsafe {
+            libc::ioctl(
+                self.0.as_raw_fd(),
+                libc::TIOCGWINSZ.into(),
```

Comment:
for this sort of thing I would typically do: `libc::TIOCGWINSZ as _` to magically infer the right type, but I'm also fine with `.into()`.

I'm curious why line 110 above doesn't need this same treatment; was it only this line that had a compile failure?

Reply:
Honestly I didn't know about `_` as a placeholder for type inference, very handy, updated to match your style.

In regards to `libc::TIOCSWINSZ` that didn't cause a compile error on this target but upon checking [some targets](https://github.com/rust-lang/libc/search?q=TIOCSWINSZ&unscoped_q=TIOCSWINSZ) also define this as `u32` so i've applied the same cast for that also.